src/h5web/explorer/EntityList.tsx

```diff
@@ -26,12 +27,11 @@ function EntityList(props: Props): ReactElement {
     <ul className={styles.group} role="group">
       {group.children.map((entity) => {
         const { uid, name } = entity;
-        const path = `${parentPath === '/' ? '' : parentPath}/${name}`;
 
         return (
           <EntityItem
             key={uid}
-            path={path}
+            path={buildEntityPath(parentPath, name)}
```

Comment:
Adding this little utility to help construct absolute paths. This will also be useful in my next PR for creating an absolute path from a relative NeXUS `defaultPath`.